rules-tests/DeadCode/Rector/FunctionLike/NarrowTooWideReturnTypeRector/Fixture/skip_non_final_inheritance.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+namespace Rector\Tests\DeadCode\Rector\FunctionLike\NarrowTooWideReturnTypeRector\Fixture;
+
+use Rector\Tests\DeadCode\Rector\FunctionLike\NarrowTooWideReturnTypeRector\Source\SomeInterface;
+use Rector\Tests\DeadCode\Rector\FunctionLike\NarrowTooWideReturnTypeRector\Sourcet\SomeAbstractClass;
+
+class SkipInheritance extends SomeAbstractClass
```

Comment:
this skipped because class is not final, please add new fixture for **final** class:

```php
final class SkipFinalClassOverrideAbstractMethod extends SomeAbstractClass
```

Reply:
This test already exists---it's called `final_inheritance`